responseheader/responseheader.go

```diff
@@ -0,0 +1,39 @@
+package responseheader
+
+import (
+	"context"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+)
+
+// UnaryServerInterceptor returns a server interceptor
+// that copies selected request metadata into response metadata.
+func UnaryServerInterceptor(allowedHeaders map[string]string) grpc.UnaryServerInterceptor {
```

Comment:
Where is the integration test?

Reply:
Added integration test for both runtime mux mw and the responseheader interceptor.